src/utils/getSchedules.ts

```diff
@@ -1,11 +1,11 @@
 import { Schedule } from '@/types/event';
 
-const isDateInRange = (selectedDate: string, startDateTime: string, endDateTime: string) => {
-  const startDate = new Date(startDateTime);
-  const endDate = new Date(endDateTime);
-  const targetDate = new Date(selectedDate);
+const isDateInRange = (selectedDate: string, start: string, end: string | null) => {
+  if (!end) {
+    return selectedDate === start;
+  }
 
-  return targetDate >= startDate && targetDate <= endDate;
+  return selectedDate >= start && selectedDate <= end;
```

Comment:
p5;
```suggestion
  return start <= selectedDate  && selectedDate <= end;
```
저는 개인적으로 값이 어떤 범위안에 포함되는지 판단할때 이렇게 쓰는게 이해하기 쉽더라구요! 알고리즘할때 이렇게 많이 썼던 기억이ㅣ...